play-ebean/src/main/java/play/db/ebean/EbeanParsedConfig.java

```diff
@@ -42,8 +49,9 @@ public Map<String, List<String>> getDatasourceModels() {
    */
   public static EbeanParsedConfig parseFromConfig(Config config) {
     Config playEbeanConfig = config.getConfig("play.ebean");
-    String defaultDatasource = playEbeanConfig.getString("defaultDatasource");
     String ebeanConfigKey = playEbeanConfig.getString("config");
+    Boolean ebeanDdlGenerateKey = playEbeanConfig.getBoolean("ddlGenerate");
+    String ebeanDefaultDatasourceKey = playEbeanConfig.getString("defaultDatasource");
```

Comment:
Please do *not* add a `...Key` suffix to the variable names here. I see you did that because of how `ebeanConfigKey` is named, but this is different, because `ebeanConfigKey` actually _really_ contains a config key, but the other variables do _not_, they contain actual values.
So please for the two variables, remove the `...Key` suffix (and rename the new var accordingly).